woocommerce/woocommerce
# Product Collection: Rename "Force Page Reload" control to "Reload Full Page" and update copy

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/52621



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add new post
2. Add Product Collection block
3. Click "create my own"
4. Open Inspector Controls > Advanced
5. **Expected:** 
    - Control is now "Reload full page"
    - Help text:
        - Option off: `Enable to enforce full page reload on certain interactions, like using paginations controls.`
        - Option on: `Browsing between pages requires a full page reload.`

6. In Product Collection insert "Featured Product" block (it's unlikely scenario and looks ugly but this block represents any 3rd party block that won't be compatible with Interactivity API)
7. Check the Inspector Controls > Advanced > "Reload full page"
8. **Expected:**
    - Option is on and disabled
    - Help Text: `Reload full page can't be disabled because there are incompatible blocks inside the Product Collection block.`

Case | Before | After
-- | -- | --
Off | <img width="264" alt="image" src="https://github.com/user-attachments/assets/0f2a30f3-8271-46db-930c-42713aa14994"> | <img width="271" alt="image" src="https://github.com/user-attachments/assets/229f73ed-211b-4fa8-a361-6f9787b18eea">
On | <img width="266" alt="image" src="https://github.com/user-attachments/assets/e9caa455-b9a5-47f5-8f59-36b92494da15"> | <img width="268" alt="image" src="https://github.com/user-attachments/assets/0bb6d891-bec0-4b37-9d63-5ba4c032e880">
On and disabled | <img width="271" alt="image" src="https://github.com/user-attachments/assets/d3c22f88-180b-4b17-a868-5cb1095f78bb"> | <img width="266" alt="image" src="https://github.com/user-attachments/assets/6d502278-e47e-4628-924f-2ef3c9b73b7b">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
